docs/presets/README.md

```diff
@@ -20,6 +20,9 @@ If emojis aren't your thing, this might catch your eye!
 [aws]
 symbol = "  "
 
+[buf]
+symbol = "🦬 "
```

Comment:
Ideally, the symbol used here would be a NerdFont symbol rather than an emoji. Do you think you can find a fitting symbol from [the cheatsheet](https://www.nerdfonts.com/cheat-sheet)?

Reply:
Okay, I think I found one that works pretty well here